Explain more auth configure-docker details (#3646)
chainguard-dev/edu: content/platform/chainctl-usage/how-to-install-chainctl.md

chainctl auth configure-docker
```

> This will:
> 
> * Create a link for `docker-credential-cgr` in the same directory as `chainctl`.
> * Update your Docker config file to call `docker-credential-cgr` when an auth token is needed. A browser window will open when the token needs to be refreshed.
> 
> {{< note >}}
> If you installed `chainctl` to a directory that needs root permissions and run as root (eg with `sudo`) when creating the `docker-credential-cgr` symlink, and then complete login in that same session, the Docker credentials will be written to `root`'s `.docker/config.json` instead of your user's Docker config. In that case, run `configure-docker` again as the correct user.
> {{< /note >}}

Reviewer: ```suggestion
This will:

* Create a link for `docker-credential-cgr` in the same directory where `chainctl` is installed.
* Update your Docker config file to call `docker-credential-cgr` when an auth token is needed. A browser window will open when the token needs to be refreshed.

{{< note >}}
If you installed `chainctl` to a directory that needs root permissions and you then create the symlink for `docker-credential-cgr` while running as root (for example, with `sudo`) , and then complete your login in that same root session, the Docker credentials will be written to the root-owned `.docker/config.json` instead of your user's Docker config.

To correct this, run `configure-docker` again as the correct user.
{{< /note >}}
```
